crates/cheatnet/tests/common/state.rs

```diff
@@ -0,0 +1,25 @@
+use camino::Utf8PathBuf;
+use cheatnet::CheatnetState;
+use include_dir::{include_dir, Dir};
+use std::path::PathBuf;
+use tempfile::{tempdir, TempDir};
+
+static PREDEPLOYED_CONTRACTS: Dir = include_dir!("crates/cheatnet/predeployed-contracts");
+
+fn load_predeployed_contracts() -> TempDir {
+    let tmp_dir = tempdir().expect("Failed to create a temporary directory");
+    PREDEPLOYED_CONTRACTS
+        .extract(&tmp_dir)
+        .expect("Failed to copy predeployed contracts to temporary directory");
+    tmp_dir
+}
```

Comment:
If this is only used for tests, can't we just use contracts at the path provided? There's no point using `include_dir!` to include contracts in the binary if this code is only for testing.

Reply:
done